kafka/consumer/base.py

```diff
@@ -21,8 +20,8 @@
 FETCH_DEFAULT_BLOCK_TIMEOUT = 1
 FETCH_MAX_WAIT_TIME = 100
 FETCH_MIN_BYTES = 4096
-FETCH_BUFFER_SIZE_BYTES = 4096
-MAX_FETCH_BUFFER_SIZE_BYTES = FETCH_BUFFER_SIZE_BYTES * 8
+FETCH_BUFFER_SIZE_BYTES = 262144
+MAX_FETCH_BUFFER_SIZE_BYTES = 157286400
```

Comment:
we have increased this ratio from 8 to 600, is this expected or may be some typo ?
Also, would you please use the previous format like `FETCH_BUFFER_SIZE_BYTES * 600`, so we can change only place.